systemd/hw-probe.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Probe hardware and upload result to Linux hardware database
+ConditionVirtualization=false
+After=local-fs.target
+
+[Service]
+Type=idle
+ExecStart=/usr/bin/hw-probe -all -upload
+IOSchedulingClass=idle
+Nice=19
+IOSchedulingClass=2
+IOSchedulingPriority=7
+ProtectSystem=full
```

Comment:
Do you have example probe uploaded by this service?

Reply:
lut 03 14:57:01 tpg-virtualbox hw-probe[12312]: Probe URL: https://linux-hardware.org/?probe=e073e88f24